### ISSUE TITLE
A11y: "Custom environment" / "Options" expand/collapse controls have wrong accessible name, do not expose state

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/desktop#1440]
### Steps To Reproduce

1. Use a screen reader/assistive technology
2. Go to the "Log in or create a new account..." view
3. Go to "Settings"
4. Set focus to the "Custom environment" expand/collapse control

### Expected Result

The control should be announced as "Custom environment" (to comply with [WCAG 2.1 SC 2.5.3 Label in Name](https://www.w3.org/TR/WCAG21/#label-in-name)), and expose its current state (whether it's expanded or collapsed)

### Actual Result

The control is announced as "Toggle visibility", and does not expose its current state

### Screenshots or Videos

![screenshot of the control in focus, and NVDA's speech viewer showing it's announced as "Toggle visibility"](https://user-images.githubusercontent.com/895831/159804556-8c8fc45f-5691-4367-a72f-6347ac80ef64.png)

### Operating System

Windows

### Operating System Version

10

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 1.32.1 Shell 16.0.7 Renderer 96.0.4664.110 Node 16.9.1 Architecture x64

**Comment by patrickhlauke** [comment]
Related https://github.com/bitwarden/browser/pull/2234

**Comment by patrickhlauke** [comment]
Same issue found for the "Options" expand/collapse in the password generator

![screenshot of password generator "Options" expand/collapse; NVDA speech viewer shows it's exposed as "Toggle Visibility", and no state indication provided](https://user-images.githubusercontent.com/895831/159808364-a0c468a2-bad0-4654-902b-d7eb3f1f47dd.png)
